src/main/deltachat/settings.js

```diff
@@ -2,6 +2,16 @@ const C = require('deltachat-node/constants')
 const log = require('../../logger').getLogger('main/deltachat/settings', true)
 
 const SplitOut = require('./splitout')
+
+const serverFlagMap = {
```

Comment:
I don't get what this map is for. Nobody uses strings like `mail_security_ssl` anywhere, they are never passed to `getConfigFor`.

What is the reason for this server flags refactoring?

Reply:
It's just for displaying the values in the Settings dialog. DC core uses the flags but humans need them to be translated into the according string values. So we get stings (like "ssl"or "plain") from the frontend and have to pass a bitmask to the dc core and vice versa